Modules/Numerics/Optimizers/src/itkExhaustiveOptimizer.cxx

```diff
@@ -25,16 +25,6 @@ namespace itk
 ExhaustiveOptimizer::ExhaustiveOptimizer()
 {
   itkDebugMacro("Constructor");
```

Comment:
Is this still useful? Could we just use the default constructor?

Reply:
@blowekamp Good question. 😃  This `itkDebugMacro("Constructor")` certainly isn't useful to me. I do see them in a few other places in ITK, though. Shall we remove them all?

If we would remove the `itkDebugMacro("Constructor")` call, we could certainly do `ExhaustiveOptimizer::ExhaustiveOptimizer() = default`.

Reply:
These sort of debug statements are useful during initial development, and occasionally during some later debugging. As this is quite a mature class by now, I vote we remove it and go with `= default;`.